src/App.tsx

```diff
@@ -1,73 +1,39 @@
 import React from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Alert } from './components/alert/Alert';
+import { Dropdown } from './components/dropdown/Dropdown';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = React.useState<Person | null>(
+    null,
+  );
+  const [appliedQuery, setAppliedQuery] = React.useState('');
+  const filteredPeople = React.useMemo(() => {
```

Comment:
Code be like:

![image](https://github.com/user-attachments/assets/8e3670e2-de55-4790-8d15-a0a714158f39)